### PR TITLE
Paging (Replace card names with images step 1)

### PR DESCRIPTION
**Pull request by alexmiddeleer**
Thinking of just skipping the unit tests for the paging lib for now. Bugs would be pretty easy to spot in the widget.

Right now all paging is client-side. I think this is fine/preferable for cards. It should be easy to upgrade the paging component to support a bubbling click event, as the paging component doesn't manage which items to show or how data is loaded.

### CODE REVIEW

File: next/components/card-list.js [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
-      ))}
-    </ul>
+    <>
+      <ul className="cardList" data-cy="cardList">
+        {cards.map((card, index) => {
+          if (!onCurrentPage(index, currentPage, pageSize)) {
+            return;
+          }
+          return (
+            <li key={card.id ? card.id : index}>
```
Comment (jtrussell) [inline next/components/card-list.js]: I think `0` is a valid card id.

File: next/components/paging-controls.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    </div>
+  );
+}
+
+PagingControls.defaultProps = {
+  pageSize: 20
+};
+
+PagingControls.propTypes = {
+  currentPage: PropTypes.any.isRequired,
```
Comment (evsheffield) [inline next/components/paging-controls.js]: I think the first three props here should be `PropTypes.number`

File: next/lib/paging.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+// 1, 20 => 20
```
Comment (evsheffield) [inline next/lib/paging.js]: I think this comment differs from the output of the function. Seems like the first page is indexed as 0, is that right?

Comment (alexmiddeleer) [reply next/lib/paging.js]: Oops, yeah it should be 0, 20 => 20. The output is not zero-based but current page is. I updated the comment.

**Pull request merged**